rpc/json/rpc.json

```diff
@@ -272,7 +272,8 @@
         },
         "types": [
             {
-                "out_point": "OutPoint object {\"tx_hash\": <tx_hash>, \"index\": <index>}."
+                "out_point": "OutPoint object {\"tx_hash\": <tx_hash>, \"index\": <index>}.",
+                "with_data": "Boolean"
```

Comment:
Should change to:

```
            {
                "out_point": "OutPoint object {\"tx_hash\": <tx_hash>, \"index\": <index>}."
            },
            {
                "with_data": "Boolean"
            }
```

Please take a look [here](https://github.com/nervosnetwork/ckb/blob/9bbd2110be80347a1db6ac8deb161a12a3b40200/devtools/doc/rpc.py#L31). This assertion will prevent your present unexpected format.
It is better to print the failed reason and what is the expected format. Could you help us to improve it?